src/gfx/2d/2d.ts

```diff
@@ -197,6 +198,7 @@ module Shumway.GFX.Canvas2D {
 
     set (state: RenderState) {
       this.clip.set(state.clip);
+      this.clipPath = state.clipPath;
```

Comment:
All other assignments in this function copy the state. Here you are aliasing the `clipPath`. Is that correct?